cardano-testnet/src/Testnet/Cardano.hs

```diff
@@ -185,30 +191,104 @@ mkTopologyConfig numNodes allPorts port True = J.encode topologyP2P
         (P2P.UseLedger DontUseLedger)
 
 cardanoTestnet :: CardanoTestnetOptions -> H.Conf -> H.Integration TestnetRuntime
-cardanoTestnet testnetOptions H.Conf {..} = do
+cardanoTestnet testnetOptions configuration@H.Conf {tempAbsPath, testnetMagic} = do
+  let tmpDir = tempAbsPath
   void $ H.note OS.os
   currentTime <- H.noteShowIO DTC.getCurrentTime
   startTime <- H.noteShow $ DTC.addUTCTime startTimeOffsetSeconds currentTime
-  configurationFile <- H.noteShow $ tempAbsPath </> "configuration.yaml"
+  let
+      numBftNodes = cardanoNumBftNodes $ cardanoNodes testnetOptions
+      bftNodesN = [1 .. numBftNodes]
+      poolNodesN = [1 .. cardanoNumPoolNodes $ cardanoNodes testnetOptions]
+      bftNodeNames = ("node-bft" <>) . show @Int <$> bftNodesN
+      poolNodeNames = ("node-pool" <>) . show @Int <$> poolNodesN
+      allNodeNames = bftNodeNames <> poolNodeNames
+
+  ( byronGenesisKeys
+    , byronDelegationCerts
+    , byronDelegationKeys) <- cardanoTestnetByronGenesis
+       (TmpPath tmpDir)
+       testnetMagic
+       testnetOptions
+       (configurationTemplate configuration)
+       startTime
+{-
+  -- dead code: not checked in any test
+  cardanoTestnetByronGenesisExpenditure testnetOptions configuration byronGenesisKeys
+  cardanoTestnetByronGovernance
+    (L.length $ cardanoBftNodeOptions testnetOptions)
+    testnetMagic
+    tmpDir
```

Comment:
Can you elaborate why you mean be "dead code".  There are a number of such references.

Reply:
Some parts of the code code call `cardano-cli` for example to create a transaction.
However that transaction is never ever transmitted to the testnet or reused.
Also the code is not tested, all the tests pass when the code is disabled.
`cardanoTestnetByronGovernance` creates genesis key, delegation certs and delegation keys,
however this are are not used.